apps/sv/frontend/src/components/governance/ProposalDetailsContent.tsx

```diff
@@ -0,0 +1,704 @@
+        gutterBottom
+      >
+        {label}
+      </Typography>
+      <Typography variant="body1" id={valueId} data-testid={valueId}>
+        {value}
+      </Typography>
+    </Box>
+  );
+};
+
+interface VoteItemProps {
+  voter: string;
+  url: string;
+  comment: string;
+  status: VoteStatus;
+  isClosed?: boolean;
+  isYou?: boolean;
+}
+
+const VoteItem = ({ voter, url, comment, status, isClosed, isYou = false }: VoteItemProps) => {
+  const getStatusColor = () => {
+    switch (status) {
+      case 'accepted':
+        return theme.palette.success.main;
+      case 'rejected':
+        return theme.palette.error.main;
+      case 'no-vote':
+        return isClosed ? theme.palette.error.main : '#ff9800';
+      default:
+        return '#757575';
+    }
+  };
+
+  const getStatusText = () => {
+    switch (status) {
+      case 'accepted':
+        return 'Accepted';
+      case 'rejected':
+        return 'Rejected';
+      case 'no-vote':
+        return isClosed ? 'No Vote' : 'Awaiting Response';
+      default:
+        return 'Unknown';
+    }
+  };
+
+  return (
+    <Box
+      sx={{
+        p: 2,
+        border: '1px solid rgba(81, 81, 81, 1)',
+        borderRadius: 2,
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'flex-start',
+      }}
+      data-testid="proposal-details-vote"
+    >
+      <Box sx={{ flexGrow: 1 }}>
+        <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
+          <PartyId
+            partyId={voter}
+            className="proposal-details-voter-party-id"
+            id="proposal-details-voter-party-id"
+          />
+          {isYou && (
+            <Chip
+              label="You"
+              size="small"
+              sx={{
+                ml: 1,
+                bgcolor: 'rgba(255, 255, 255, 0.1)',
+              }}
+              data-testid="proposal-details-your-vote-chip"
+            />
+          )}
+        </Box>
+        {comment && (
+          <Typography variant="body2" color="text.secondary">
+            {comment}
+          </Typography>
+        )}
+        {url && (
+          <Typography variant="body2" color="text.secondary">
+            <Link href={sanitizeUrl(url)} target="_blank" color="primary">
+              {sanitizeUrl(url)}
+            </Link>
+          </Typography>
+        )}
+      </Box>
+      <Box sx={{ display: 'flex', alignItems: 'center' }}>
+        <Box
+          component="span"
+          sx={{
+            display: 'inline-block',
+            width: 16,
+            height: 16,
+            borderRadius: '50%',
+            bgcolor: getStatusColor(),
+            mr: 1,
+          }}
+        />
+        <Typography
+          variant="body2"
+          color={getStatusColor()}
+          data-testid="proposal-details-vote-status-value"
+        >
+          {getStatusText()}
+        </Typography>
+      </Box>
+    </Box>
+  );
+};
+
+interface OffboardMemberSectionProps {
```

Comment:
This file has gotten quite big so i'll most likely move the types from this point before to new files in my next refactor(probably next pr)